api/RTCPeerConnection.json

```diff
@@ -274,10 +274,12 @@
               }
             ],
             "safari": {
-              "version_added": null
+              "version_added": "11",
```

Comment:
Yeah, a bunch of things seem to have been shipped in Safari 11 but then removed in 12, because there was no agreement on the spec side that these things should be defined.